cinema/models.py

```diff
@@ -60,6 +60,10 @@ class Meta:
     def __str__(self):
         return self.movie.title + " " + str(self.show_time)
 
+    @property
+    def tickets_available(self):
+        return self.cinema_hall.capacity - self.tickets.count()
```

Comment:
The `tickets_available` property in the `MovieSession` model calculates the available tickets by subtracting the count of tickets from the cinema hall's capacity. Ensure that the `related_name` for the `tickets` field in the `Ticket` model is correctly set to allow this reverse lookup.